api/deploy/ocp/scc.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2021 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ocp
+
+import (
+	"bytes"
+	_ "embed"
+	"text/template"
+
+	"github.com/ghodss/yaml"
+	secv1 "github.com/openshift/api/security/v1"
+)
+
+//go:embed scc.yaml
+var securityContextConstraints string
+
+type SecurityContextConstraintsValues struct {
+	Namespace string
+	Prefix    string
+}
+
+var SecurityContextConstraintsDefaults = SecurityContextConstraintsValues{
+	Namespace: "ceph-csi",
+	Prefix:    "",
+}
+
+// NewSecurityContextConstraints takes a namespace and the name of the deployer
+// (like Rook) and returns a SecurityContextConstraints object that can be
+// deployed on OpenShift.
+//
+// The deployer parameter (when not an empty string) is used as a prefix for
```

Comment:
Can we clarify the deployer parameter? Especially if we want to include an `-` in it? Right now it's unclear how the `user` in the SCC will turn into.

Reply:
Right. I'll make it so that if no `deployer` is passed, the prefix is empty, otherwise the prefix will be set to `<deployer>-` with the `-`.